locales/en.json

```diff
@@ -1,15 +1,15 @@
 {
 	"Collaborative markdown notes": "Collaborative markdown notes",
-	"Realtime collaborative markdown notes on all platforms.": "Realtime collaborative markdown notes on all platforms.",
-	"Best way to write and share your knowledge in markdown.": "Best way to write and share your knowledge in markdown.",
+	"Realtime collaborative markdown notes on all platforms.": "Callaborate on markdown notes on all platforms.",
+	"Best way to write and share your knowledge in markdown.": "The best way to write and share your knowledge in markdown.",
```

Comment:
I knew there should have "The" in front of this sentence.
Because we already have the logo and "HackMD" above this.
So "The" is omitted on purpose.